addon/styles/addon.css

```diff
@@ -61,3 +70,27 @@
   /*top: 100%;*/
   bottom: 0;
 }
+
+.-horizontal .dragSortItem.-placeholderAbove:before,
+.-horizontal .dragSortItem.-placeholderBelow:before {
+  text-align: center;
+  width: 25px;
+  top: 0;
```

Comment:
`top: 0` fails to center the placeholder arrow vertically.

This should do vertical centering:

```css
.-horizontal .dragSortItem.-placeholderAbove:before,
.-horizontal .dragSortItem.-placeholderBelow:before {
  top: 50%;
  transform: translateY(-50%);
}
```